### PR TITLE
Wishlist page

### PR DESCRIPTION
**Pull request by amartya-rs**
wishlist page of the ecom site

### CODE REVIEW

**Review by zoef-yt — APPROVED** [review]
Good code. Good to merge.

File: pages/wishlist-page/wishlist.css [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    flex-wrap: wrap;
+    flex-direction: column;
+    align-items: center;
+}
+
+
+/*wishlish section*/
+.wishlist-items {
+    max-width: 85%;
+
```
Comment (developer-anand) [inline pages/wishlist-page/wishlist.css:28]: remove spacing.

**Review by developer-anand — APPROVED** [review]
Good to merge.

File: pages/wishlist-page/wishlist.css (hunk 1 above)
Comment (amartya-rs) [reply pages/wishlist-page/wishlist.css:28]: Will take care of spacing. Thanks for your feedback.

**Review by srishtigautam21 — APPROVED** [review]
Good work...good to merge

**Pull request merged**